python/lsst/ap/association/packageAlerts.py

```diff
@@ -63,6 +67,18 @@ class PackageAlertsConfig(pexConfig.Config):
         default=os.path.join(os.getcwd(), "alerts"),
```

Comment:
Oh dear, I just noticed this: we definitely don't want it to default to writing to the current working directory, especially since we're using it as the dumping ground for alerts that fail to be sent via kafka! I think this path should either have no default, or be a tempdir.